openshift-tools/pkg/discover-etcd-initial-cluster/initial-cluster.go

```diff
@@ -198,15 +198,15 @@ func (o *DiscoverEtcdInitialClusterOptions) Run() error {
 		// this is the latter. Do not let etcd start and report the condition as an error.
 		// Result: return error and restart container
 		if found && !dataDirExists {
-			return fmt.Errorf("member %q dataDir has been destoyed and must be removed from the cluster", target.String())
+			return fmt.Errorf("member %q dataDir has been destroyed and must be removed from the cluster", target.String())
 		}
 
 		// Condition: member not found with dataDir
-		// The member has been removed from the cluster. etcd should not start and return error. The dataDir will be archived once the operator
+		// The member has been removed from the cluster. The dataDir will be archived once the operator
 		// scales up etcd.
-		// Result: return error and restart container
+		// Result: retry member check allowing operator time to scale up etcd again on this node.
 		if !found && dataDirExists {
-			return fmt.Errorf("member %q is no longer a member of the cluster and should not start", target.String())
+			fmt.Fprintf(os.Stderr, "      member %q not found in member list but dataDir exists, check operator logs for possible scaling problems\n", target.String())
 		}
 
```

Comment:
Before your recent fix, we used to return with success if dataDir was present:
https://github.com/openshift/etcd/commit/af729a8c3a354aa7eee3e27eb10658173fda613d#diff-3d1414e0d47047e4fdcae958fb7654d082c1eb1b71cf4eb38aeb3190db678208L168-R172

Reply:
Yeah in the eyes of the operator this is essentially scaling up workflow so it feels reasonable that we give the container a bit longer to be observable.